app/scripts/lib/state-corruption/state-corruption-recovery.test.ts

```diff
@@ -84,7 +84,7 @@ describe('CorruptionHandler.handleStateCorruptionError', () => {
                   method: '__INVALID__',
                 },
               });
-            } else if (message.data.method === 'RELOAD') {
+            } else if (message.data.method === 'RELOAD_WINDOW') {
```

Comment:
renamed because it was confusing me what it did, as we also use `browser.runtime.reload`, which this does NOT invoke (it invokes `window.location.reload();`)